templates/vue/src/components/modals/UserAnswersModal/index.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <b-modal
+    id="user-answers-modal"
+    class="xl-modal"
+    data-qa="user-answers-modal"
+    :visible="show"
+    size="xl"
+    title="User Answers"
+    scrollable
+    body-class="p-0"
+    @hidden="$emit('close')"
+  >
+    <b-container fluid class="px-0">
+      <b-tabs card>
+        <b-tab
+          title="Answers"
+          :active="tab === 'answers'"
+          @click="$emit('change:tab', 'answers')"
+        ></b-tab>
+        <user-answers-form />
+      </b-tabs>
+    </b-container>
+    <template slot="modal-footer">
+      <b-button size="sm" variant="secondary" @click="closeModal">
+        Cancel
+      </b-button>
+    </template>
+  </b-modal>
+</template>
+
+<script>
+import DragSelectModular from "@/utils/dragSelectModular"
+import UserAnswersForm from "./UserAnswersForm"
+export default {
+  name: "user-answers-modal",
+  components: {
+    UserAnswersForm,
+  },
+  props: {
+    show: {
+      type: Boolean,
+      required: true,
+    },
+    tab: {
+      type: String,
+      required: false,
+      default: "",
+    },
+  },
+  mounted() {
+    this.$root.$on("bv::modal::show", (_, modalId) => {
+      if (modalId === "user-answers-modal") {
+        DragSelectModular.removeDragSelectListener()
+      }
+    })
+    this.$root.$on("bv::modal::hide", (_, modalId) => {
+      if (modalId === "user-answers-modal") {
+        DragSelectModular.addDragSelectListener()
+        this.$emit("close")
+      }
+    })
+  },
+  methods: {
+    closeModal() {
+      this.$emit("close")
+    },
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+#save-button {

```

Comment:
I couldn't find an element with this id in the component, so I think this CSS can be deleted.